tests/sentry/backup/test_dependencies.py

```diff
@@ -0,0 +1,55 @@
+from difflib import unified_diff
+
+from sentry.backup.dependencies import DependenciesJSONEncoder, dependencies, sorted_dependencies
+from sentry.testutils.factories import get_fixture_path
+
+encoder = DependenciesJSONEncoder(
+    sort_keys=True,
+    ensure_ascii=True,
+    check_circular=True,
+    allow_nan=True,
+    indent=2,
+    encoding="utf-8",
+)
+
+
+def test_detailed():
```

Comment:
is it worth to use `insta_snapshot`?

Reply:
Discussed offline, decided that the value of these being in pure JSON made this okay for now. If its still a problem going forward, it is easy to convert these to use insta snapshot instead.